mcy/src/pages/News.jsx

```diff
@@ -1,5 +1,12 @@
+import Header from "../components/Layout/Header";
+
 const News = () => {
-  return <div>News</div>;
+  return (
+    <div>
+      <Header />
+      <div>News</div>
+    </div>
```

Comment:
```suggestion
    <Layout>
      <div>News</div>
    </Layout>
```
레이아웃 안에 컨텐츠를 표시하는 방식으로 다 변경 필요